Doppler.ContactPolicies.Api.Test/GetContactPoliciesSettingsTest.cs

```diff
@@ -74,7 +74,6 @@ public async Task
 
             var contactPoliciesMock = new Mock<IContactPoliciesService>();
             contactPoliciesMock.Setup(x => x.GetIdUserByAccountName(accountName)).ReturnsAsync(expectedNotFoundedUserId);
-            contactPoliciesMock.Setup(x => x.GetContactPoliciesSettingsAsync(accountName)).ReturnsAsync(expected);
```

Comment:
Maybe we can rename the `expectedNotFoundedUserId` as `expectedNotFoundedIdUser`

Reply:
👍